### PR TITLE
Preliminary Pull Request

### PR DESCRIPTION
**Pull request by fschopp**
Documentation:
- Linear/logistic now mention condition number
- k-Means: General update, better references

Logistic regression:
- Error message in case of numerical issues.

Misc:
- Updated Read-Me

_Please do not merge, yet. I'm putting this out for review first._

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -164,6 +164,20 @@ if(NOT M4_BINARY)
...
+    COMPONENT core
+    PATTERN ".DS_Store" EXCLUDE
+)
+install(
+    FILES
+        "${CMAKE_CURRENT_SOURCE_DIR}/ReadMe.txt"
+        "${CMAKE_CURRENT_SOURCE_DIR}/ReleaseNotes.txt"
+    DESTINATION doc
+    COMPONENT core
+)
```
Comment (cwelton) [inline CMakeLists.txt:181]: Installation of license files, readme and release notes.  Good!

**Comment by cwelton** [comment]
Preliminary pull request looks good.

Still need updated Release notes.  Is there anything else that you are trying to update at this stage?  What is the planned delta for getting from Preliminary to Final PR?

**Comment by fschopp** [comment]
Yes, I still need to write a few lines for the probability module. If you or somebody else has the time to summarize and beautify the release notes, that would be fantastic.

**Comment by cwelton** [comment]
Review completed

**Pull request merged**